core/project_settings.h

```diff
@@ -35,6 +35,8 @@
 #include "core/os/thread_safe.h"
 #include "core/set.h"
 
+static const String IMPORTED_FILES_PATH = "res://.godot/imported";
```

Comment:
This should be a class constant, not a global scope one.

Reply:
Done. Note that this takes up two lines (3 incl. spacing) because we can't have in-class initializers for String.